app/__init__.py

```diff
@@ -80,11 +85,16 @@ def shutdown_session(exception=None):
         db.session.remove()
 
 
+from config import config_dict
+
+DEBUG = db_config("DEBUG", default=True)
+get_config_mode = "Development" if DEBUG else "Production"
```

Comment:
is this a duplicate of line 56 and 57 above?

Reply:
Yeah, I have removed it